.changeset/healthy-lies-decide.md

```diff
@@ -0,0 +1,11 @@
+---
+'@emotion/serialize': patch
+'@emotion/styled': patch
+'@emotion/cache': patch
+'@emotion/utils': patch
+'@emotion/css': patch
+'@emotion/react': patch
+'@emotion/babel-plugin': patch
```

Comment:
I'm releasing extra packages than necessary just so people can more easily can bump the thing they directly depend on rather than refreshing whole lock files.